01-Login/src/services/auth.service.ts

```diff
@@ -1,11 +1,14 @@
 import { Injectable, NgZone } from '@angular/core';
 import { Storage } from '@ionic/storage';
+import { SafariViewController } from '@ionic-native/safari-view-controller';
 
 // Import AUTH_CONFIG, Auth0Cordova, and auth0.js
 import { AUTH_CONFIG } from './auth.config';
 import Auth0Cordova from '@auth0/cordova';
 import * as auth0 from 'auth0-js';
 
+declare let cordova: any;
```

Comment:
This is being declared here but never seems to get assigned to anything. However, it's used later:

> `cordova.InAppBrowser.open(url, '_system');`

I would assume this would produce some kind of runtime error when this code is run.

Reply:
Also, is there a reason why this can't be a class member within `AuthService`?